### ISSUE TITLE
Uncaught TypeError: Cannot read properties of undefined (reading 'RΦ')

### ISSUE THREAD

**Issue by khusamov** [typescript-rtti/typescript-rtti#48]
The following example does not work on Rollup.js

```typescript
import "reflect-metadata"
import {reflect, CallSite} from 'typescript-rtti'

function foo<T>(num : number, callSite? : CallSite) {
	if (!callSite) throw 'callSite is undefined'
	console.log(
		reflect(callSite)
			.typeParameters[0]
			.isClass(Boolean),
		// => true

		reflect(callSite)
			.parameters[0]
			.isClass(Number),
		// => true

		reflect(callSite)
			.parameters[0]
			.is('literal'),
		// => true

		reflect(callSite)
			.parameters[0].as('literal')
			.value
		// => 123
	)
}

foo<Boolean>(123);
```

Error text:

```
index.js:18 Uncaught TypeError: Cannot read properties of undefined (reading 'RΦ')
    at Object.a (index.js:18:1)
    at foo (index.ts:17:20)
    at index.ts:39:1
```

**Comment by rezonant** [comment]
Works in the playground, so it must be rollup-specific: https://typescript-rtti.org/#aW1wb3J0IHtyZWZsZWN0LCBDYWxsU2l0ZX0gZnJvbSAndHlwZXNjcmlwdC1ydHRpJwoKZnVuY3Rpb24gZm9vPFQ+KG51bSA6IG51bWJlciwgY2FsbFNpdGU/IDogQ2FsbFNpdGUpIHsKCWlmICghY2FsbFNpdGUpIHRocm93ICdjYWxsU2l0ZSBpcyB1bmRlZmluZWQnCgljb25zb2xlLmxvZygKCQlyZWZsZWN0KGNhbGxTaXRlKQoJCQkudHlwZVBhcmFtZXRlcnNbMF0KCQkJLmlzQ2xhc3MoQm9vbGVhbiksCgkJLy8gPT4gdHJ1ZQoKCQlyZWZsZWN0KGNhbGxTaXRlKQoJCQkucGFyYW1ldGVyc1swXQoJCQkuaXNDbGFzcyhOdW1iZXIpLAoJCS8vID0+IHRydWUKCgkJcmVmbGVjdChjYWxsU2l0ZSkKCQkJLnBhcmFtZXRlcnNbMF0KCQkJLmlzKCdsaXRlcmFsJyksCgkJLy8gPT4gdHJ1ZQoKCQlyZWZsZWN0KGNhbGxTaXRlKQoJCQkucGFyYW1ldGVyc1swXS5hcygnbGl0ZXJhbCcpCgkJCS52YWx1ZQoJCS8vID0+IDEyMwoJKQp9Cgpmb288Qm9vbGVhbj4oMTIzKTs=

**Comment by rezonant** [comment]
Reproduction: https://github.com/typescript-rtti/typescript-rtti-issue-48

**Comment by rezonant** [comment]
Gist with emitted bundle: https://gist.github.com/rezonant/e0f9289538bd370496988f877c08b025

**Comment by rezonant** [comment]
Comes down to this: Here, the `CallSite` interface does not have an emitted token, so `dist["IΦCallSite"]` will be undefined. This isn't likely specific to rollup, as it turns out.

```
[91]: { LΦ: t => dist["IΦCallSite"] }
```

It works in the playground because the actual `typescript-rtti` library code is not reachable by TS, so an `T_ANY` (`~`) type reference is emitted in its place:

```
[79]: { RΦ: t => ({ TΦ: "~" }) }
```

This would not be true outside of the Playground.

**Comment by rezonant** [comment]
The fix will be released as 0.5.6 once the corpus test suite passes
